gwpy/signal/tests/test_window.py

```diff
@@ -29,7 +29,7 @@
 
 
 def test_canonical_name():
-    assert window.canonical_name('Hanning') == 'hann'
+    assert window.canonical_name('Han') == 'hann'
```

Comment:
Typo?

Reply:
No, this is an explicit test of the ability for the `canonical_name()` function to resolve the word `'Han'` to the window _function_ `hann()`.

Context: scipy allows for 'equivalencies' between different names to allow multiple name variants to resolve to the same window function, e.g. https://github.com/scipy/scipy/blob/0bd1607dcaab6e8345f39662e52c87a87564ed7a/scipy/signal/windows/_windows.py#L2109.